### PR TITLE
Remove wrong tag

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/Serilog/Core/Enrichers/PropertyEnricher.cs [hunk 1]
```diff
@@ -32,7 +32,6 @@ public class PropertyEnricher : ILogEventEnricher
         /// </summary>
         /// <param name="name">The name of the property.</param>
         /// <param name="value">The value of the property.</param>
-        /// <returns>A handle to later remove the property from the context.</returns>
         /// <param name="destructureObjects">If true, and the value is a non-primitive, non-array type,
         /// then the value will be converted to a structure; otherwise, unknown types will
         /// be converted to scalars, which are generally stored as strings.</param>
```
Comment (bartelink) [inline src/Serilog/Core/Enrichers/PropertyEnricher.cs:37]: is L39 superfluous too?

Comment (sungam3r) [reply src/Serilog/Core/Enrichers/PropertyEnricher.cs:37]: EnsureValidName throws this exception

Comment (bartelink) [reply src/Serilog/Core/Enrichers/PropertyEnricher.cs:37]: Apologies; L38 in your version, i.e. wondering (without having it in an IDE) whether `/// <returns></returns>` is necessary before the exception list (there isn't one before the method after, but perhaps it is necessary for a ctor)

Comment (sungam3r) [reply src/Serilog/Core/Enrichers/PropertyEnricher.cs:37]: ooops, fixed

**Comment by nblumhardt** [comment]
Thanks @sungam3r, looks good!

**Pull request merged**